### ISSUE TITLE
accessibility - manage text size does not work on some sites

### ISSUE THREAD

**Issue by zoff99** [duckduckgo/Android#6542]
### Describe the bug

the text size is tiny

visit https://blog.fefe.de/ (now with no articles since fefe did not post for months) and the text size is tiny
if you visit: https://blog.fefe.de/?mon=202505 the text size if fine

### How to Reproduce

1. in the settings set: 170% on the size slider. check "manage text size"
2. visit https://blog.fefe.de/
3. the text is super small

### Expected behavior

all text sizes should be larger

### Environment

```markdown
- DDG App Version: 5.241.1
```

**Comment by cmonfortep** [comment]
Thanks for reaching out. This seems to be related to the font size set by the website itself. We apply a multiplier based on the original font size. In this case, the feature is working as expected. The font size is increased by the specified percentage, but since the original size is very small, the result still appears small.

**Comment by zoff99** [comment]
@cmonfortep are you sure? the same blog with blog articles looks fine, without articles look tiny.
is there a setting for minimum font size, like other browser do have?
the current behavour is bad for people with bad eye sight.

**Comment by cmonfortep** [comment]
Yes, I agree looks not good, but it's the website size the problem here. We offer increase font size by %, but not an absolute value since we need to increase all text sizes by the same multiplier in order to keep it consistent when site use different fonts and font sizes. What we could do is to increase the % we can apply, but we found 170% working well on sites.

**Comment by zoff99** [comment]
chromium has a minimum font size setting, firefox also has a "Minimum font size" setting

**Comment by cmonfortep** [comment]
@zoff99 I'm trying to find those settings inside Firefox and chrome on their Android apps so I can see how it works and share this as feedback internally. However, I don't see those options in their accessibility settings page. Could you indicate me where they are?

Just for context, we attend bugs in the GitHub repository, and as mentioned, the feature is working as expected. I'm noting this as feedback/feature request, and I will share it internally with product team.
